.github/workflows/main.yml

```diff
@@ -313,10 +313,19 @@ jobs:
       # the coverage reports out of the container and into the host filesystem.
       - name: Copy coverage reports to the host
         run: ./tools/copy_docker_coverage_reports.sh test_container
-      - name: Upload coverage reports to Codecov
+      - name: Upload Main coverage report to Codecov
         uses: codecov/codecov-action@v5.3.1
         with:
           token: ${{ secrets.CODECOV_TOKEN }}
+          exclude: apps/prairielearn/python
+          flags: javascript
+
+      - name: Upload Python coverage report to Codecov
```

Comment:
Can we use the `paths:` config option in individual flags in `.codecov.yml` instead of uploading multiple times with different flags?

Reply:
It's unclear that this workflow would be possible: https://github.com/codecov/example-flags/pull/3/files#diff-3e878d9bdc47f29a0ab909a7db939acf08d2d2f1aaba81e6f897b32361fa40db

describes both, 

https://docs.codecov.com/docs/flags#step-1-flag-creation-in-uploader

> To apply a flag you must submit coverage reports with the -F flagname included in the upload command. (Note: remember, Codecov always recommends the [Uploader in your CI](https://docs.codecov.com/docs/codecov-uploader))

> Once the uploader step is done (above), Flag-level coverage is being sent to Codecov, but, to unlock the full feature set of Codecov, you'll additionally to add Flag Management to your repository-level YAML.

describes both

Reply:
It's unclear that this workflow would be possible: https://github.com/codecov/example-flags/pull/3/files#diff-3e878d9bdc47f29a0ab909a7db939acf08d2d2f1aaba81e6f897b32361fa40db

describes both, 

https://docs.codecov.com/docs/flags#step-1-flag-creation-in-uploader

> To apply a flag you must submit coverage reports with the -F flagname included in the upload command. (Note: remember, Codecov always recommends the [Uploader in your CI](https://docs.codecov.com/docs/codecov-uploader))

> Once the uploader step is done (above), Flag-level coverage is being sent to Codecov, but, to unlock the full feature set of Codecov, you'll additionally to add Flag Management to your repository-level YAML.

describes both

Reply:
Gotcha, thanks for pointing that out. I continue to find a lot of Codecov's design decisions to be mystifying 😅

Reply:
Agreed...